src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -396,6 +399,31 @@ AmendmentTableImpl::doValidation (
     return amendments;
 }
 
+std::vector <uint256>
+AmendmentTableImpl::getDesired ()
+{
```

Comment:
Great suggestion, Brad. I pushed a commit with the change. It's a small efficiency tradeoff for less code duplication. @JoelKatz thoughts?